kroxylicious-systemtests/src/test/resources/kafka-admin-client-template.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: %NAME%
+  labels:
+    app: %NAME%
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: %NAME%
+  template:
+    metadata:
+      labels:
+        app: %NAME%
+      name: %NAME%
+      namespace: %NAMESPACE%
+    spec:
+      containers:
+        - name: admin
+          image: quay.io/strimzi-test-clients/test-clients:latest-kafka-%KAFKA_VERSION%
+          command: [ "admin-client" ]
+          args: [ "%ARGS%" ]
```

Comment:
Passing an array like this is weak.   If one of the arguments were to pass a character significant to yaml, it'd fail.

Reply:
File removed